package-lock.json

```diff
@@ -5405,8 +5326,7 @@
         "native-promise-only": {
             "version": "0.8.1",
             "resolved": "https://registry.npmjs.org/native-promise-only/-/native-promise-only-0.8.1.tgz",
-            "integrity": "sha1-IKMYwwy0X3H+et+/eyHJnBRy7xE=",
-            "dev": true
```

Comment:
@svnv is it correct to include `native-promise-only` as a dependency to the module?

Reply:
Added by dc02492f